Makefile

```diff
@@ -38,10 +38,9 @@ dev:
 docs:
 	$(MAKE) -C ./docs
 
-fix:
-	$(PYTHON) -m ruff check --fix
-
 fmt:
+	$(PYTHON) -m autoflake --remove-all-unused-imports --in-place --recursive .
```

Comment:
`ruff check --fix` will also remove unused imports; if you only want to sort imports and remove unused, you can run `ruff check --select I,F401 --fix`. (`F401` is the code for unused imports, and `I` is the code group for isort-related issues).

Reply:
haha, I also just realized that. I had forgotten Ruff is a drop-in replacement for all of these tools.